openquake/qa_tests_data/classical/case_19/gmpe_logic_tree.xml

```diff
@@ -27,25 +27,17 @@
         <logicTreeBranchingLevel branchingLevelID="bl2">
             <logicTreeBranchSet uncertaintyType="gmpeModel" branchSetID="bs2" applyToTectonicRegionType="Stable Shallow Crust">
 
-                <logicTreeBranch branchID="b21"> <uncertaintyModel>AkkarBommer2010</uncertaintyModel>
-                    <uncertaintyWeight>0.20</uncertaintyWeight>
-                </logicTreeBranch>
-                
-                <logicTreeBranch branchID="b22"> <uncertaintyModel>CauzziFaccioli2008</uncertaintyModel>
-                    <uncertaintyWeight>0.20</uncertaintyWeight>
-                </logicTreeBranch>
-                
-                <logicTreeBranch branchID="b23"> <uncertaintyModel>ChiouYoungs2008</uncertaintyModel>
-                    <uncertaintyWeight>0.20</uncertaintyWeight>
-                </logicTreeBranch>
-                
-                <logicTreeBranch branchID="b24"> <uncertaintyModel>ToroEtAl2002SHARE</uncertaintyModel>
-                    <uncertaintyWeight>0.20</uncertaintyWeight>
-                </logicTreeBranch>
-                
-                <logicTreeBranch branchID="b25"> <uncertaintyModel>Campbell2003SHARE</uncertaintyModel>
-                    <uncertaintyWeight>0.20</uncertaintyWeight>
-                </logicTreeBranch>
+              <logicTreeBranch branchID="b2">
```

Comment:
Why are you changing an existing test? Wouldn't it be better to add a new one?

Reply:
The interesting thing is that in this test using the AvgGMPE produces the *same mean curves* as before, up to the numeric precision, even if in general the AvgGMPE does NOT produce the same mean curves. I am reusing an existing tests because I wanted to check the difference between the hazard curves obtained from the original logic tree and from the collapsed one.